### PR TITLE
fix endless spinner

### PR DESCRIPTION
**Pull request by patrickweh**
fix sort not working properly

### CODE REVIEW

File: tests/Feature/Web/OrderPositionsTest.php [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+use FluxErp\Models\Permission;
+use FluxErp\Models\PriceList;
+use FluxErp\Tests\Feature\BaseSetup;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class OrderPositionsTest extends BaseSetup
+{
+    use DatabaseTransactions;
+
+    private Order $order;
```
Comment (SirSplasch) [inline tests/Feature/Web/OrderPositionsTest.php:23]: property can be removed

File: src/Console/Commands/MakeWidget.php [hunk 2]
```diff
@@ -29,16 +29,16 @@ public function handle(): void
     {
         $name = $this->argument('name');
         $className = Str::studly($name);
-        $snakeCaseClassName = Str::snake($className);
+        $kebabCaseClassName = Str::kebab($className);
 
         $stub = $this->getStubContents();
-        $stub = str_replace(['{{class}}', '{{snake_case class}}'], [$className, $snakeCaseClassName], $stub);
+        $stub = str_replace(['{{class}}', '{{snake_case class}}'], [$className, $kebabCaseClassName], $stub);
```
Comment (SirSplasch) [inline src/Console/Commands/MakeWidget.php:35]: This is confusing, text is snake_case class, and it gets replaced with kebab-case

File: lang/de.json [hunk 3]
```diff
@@ -747,5 +747,7 @@
     "product.{id}.get": "Produkte anzeigen",
     "profiles.{id?}.get": "Benutzer verwalten",
     "serial-numbers.get": "Seriennummern anzeigen",
-    "service.{serialnumberid?}.get": "Serviceanfrage erstellen"
+    "service.{serialnumberid?}.get": "Serviceanfrage erstellen",
+    "Order.order Number": "Auftragsnummer",
```
Comment (SirSplasch) [inline lang/de.json:751]: where is this key from, this doesn't seem right.

**Review by SirSplasch — CHANGES_REQUESTED** [review]

File: src/Livewire/DataTables/OrderPositionList.php [hunk 4]
```diff
@@ -38,84 +40,10 @@ public function mount(): void
...
-            'name' => 'product_number',
-        ];
-    }
-
-    public function getRightAppends(): array
-    {
-        return [
-            'name' => 'alternative_tag',
-        ];
+        return $builder->orderBy('order_id');
```
[the remark this answers is not in the document]
Comment (patrickweh) [reply src/Livewire/DataTables/OrderPositionList.php:47]: eager loaded by new datatable version

File: lang/de.json (hunk 3 above)
Comment (patrickweh) [reply lang/de.json:751]: generated from Headline:: on column name

**Review by SirSplasch — APPROVED** [review]

**Pull request merged**